dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/job/BaseMessageManager.java

```diff
@@ -27,52 +27,19 @@
  */
 package org.hisp.dhis.tracker.imports.job;
 
-import java.io.IOException;
-import javax.jms.JMSException;
-import javax.jms.TextMessage;
-import org.hisp.dhis.artemis.MessageManager;
+import lombok.RequiredArgsConstructor;
 import org.hisp.dhis.common.AsyncTaskExecutor;
-import org.hisp.dhis.common.CodeGenerator;
-import org.hisp.dhis.render.RenderService;
 import org.springframework.stereotype.Component;
 
 /**
  * @author Zubair Asghar
  */
 @Component
+@RequiredArgsConstructor
 public abstract class BaseMessageManager {
-  private final MessageManager messageManager;
-
   private final AsyncTaskExecutor taskExecutor;
 
-  private final RenderService renderService;
-
-  public BaseMessageManager(
-      MessageManager messageManager, AsyncTaskExecutor taskExecutor, RenderService renderService) {
-    this.messageManager = messageManager;
-    this.taskExecutor = taskExecutor;
-    this.renderService = renderService;
-  }
-
-  public String addJob(TrackerSideEffectDataBundle sideEffectDataBundle) {
-    String jobId = CodeGenerator.generateUid();
-    sideEffectDataBundle.setJobId(jobId);
-
-    messageManager.sendQueue(getTopic(), sideEffectDataBundle);
-
-    return jobId;
-  }
-
   public void executeJob(Runnable runnable) {
     taskExecutor.executeTask(runnable);
```

Comment:
Remove/inline the BaseMessageManager as it just delegates to the AsyncTaskExecutor without adding any other logic.

Reply:
Fixed